ui/app/routes/vault/cluster/oidc-provider.js

```diff
@@ -148,7 +148,8 @@ export default class VaultClusterOidcProviderRoute extends Route {
     } catch (errorRes) {
       const resp = await errorRes.json();
       const code = resp.error;
-      if (code === 'max_age_violation' || resp?.errors?.includes('permission denied')) {
+      const permissionDenied = resp?.errors?.some((str) => str.includes('permission denied'));
```

Comment:
Maybe add a comment here to clarify it's important we check a partial match and not an exact match?
```suggestion
// We're using a partial match ("includes") to catch any occurrence of "permission denied" in error messages.
// This ensures we catch permissions errors in all workflows, even if the message is not an exact match.
      const permissionDenied = resp?.errors?.some((str) => str.includes('permission denied'));
```